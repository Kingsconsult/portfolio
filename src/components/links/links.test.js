import React from 'react';
import renderer from "react-test-renderer";
import Links  from "./index";
 
import { ThemeProvider } from "styled-components";
const theme = {
  mode: "light"
};


it("links renders correctly when called ", () => {
  const tree = renderer
    .create(
      <ThemeProvider theme={theme}>
            <Links />
      </ThemeProvider>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});