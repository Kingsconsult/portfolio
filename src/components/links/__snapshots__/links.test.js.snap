// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`cards renders correctly when called  1`] = `
<a
  className="sc-AxirZ iQxfjs"
  size="nav"
/>
`;

exports[`links renders correctly when called  1`] = `
<a
  className="sc-AxirZ iQxfjs"
  size="nav"
/>
`;
